scripts/vagrantCloudBasedContainers.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -ex
```

Comment:
There should be at least a link to the documentation and JIRA ticket here, and preferably a single-sentence about what the script is meant to do.